langchain4j-redis/src/main/java/dev/langchain4j/store/embedding/redis/RedisChatMemoryStore.java

```diff
@@ -0,0 +1,64 @@
+package dev.langchain4j.store.embedding.redis;
```

Comment:
Should this be in `package dev.langchain4j.store.memory.chat;` ?

Reply:
changed to `dev.langchain4j.store.memory.chat.redis` as seen in CassandraChatMemoryStore